src/App.tsx

```diff
@@ -334,8 +343,12 @@ function App() {
                   <SelectItem value="US States" className="text-white hover:bg-slate-700">US States</SelectItem>
                 </SelectContent>
               </Select>
-              <span className="text-sm">Score: <span className="text-emerald-400 font-bold">{score}</span></span>
-              <span className="text-sm">Games: <span className="text-emerald-400 font-bold">{gamesPlayed}</span></span>
+              <span className="text-sm">
+                {!isMobile && 'Score: '}<span className="text-emerald-400 font-bold">{score}</span>
+              </span>
+              <span className="text-sm">
+                {!isMobile && 'Games: '}<span className="text-emerald-400 font-bold">{gamesPlayed}</span>
+              </span>
```

Comment:
Instead of hiding the title, just hide the whole `Games` span on mobile, and only show Score: {num}